docs/source/tutorials/a_deeper_look.rst

```diff
@@ -18,7 +18,7 @@ At this point, you have just executed the following command.
 
 Let's dissect what just happened step by step.  First, this command executes
 manim on the file ``scene.py``, which contains our animation code.  Further,
-this command tells manim exactly which ``Scene`` to be rendered, in this case
+this command tells manim exactly which ``Scene`` to be rendered, in this case,
```

Comment:
too many pauses imo.

Reply:
Won’t hurt to have them, right?

Reply:
also missing "is". Which scene is to be rendered.

alternative "Which scene to render"

Reply:
added "is"